helpers.php

```diff
@@ -142,3 +142,25 @@ function include_template($name, array $data = []) {
 
     return $result;
 }
+
+function get_database_connection() {
+$connection = mysqli_connect("localhost", "root", "", "affairs_ok" );
+    if($connection === false) {
+        printf("Соединение не установлено",mysqli_connect_error());
```

Comment:
почему здесь используется printf? Он используется обычно для форматированного вывода. Например, в данном случае его можно было бы использовать так: 
```suggestion
        printf("Соединение не установлено %s",mysqli_connect_error());
```

А если так не нужно, тогда просто print с конкатинацией.

Reply:
исправлено